rust_snuba/src/strategies/processor.rs

```diff
@@ -214,9 +244,21 @@ impl<TResult: Clone, TNext: Clone> MessageProcessor<TResult, TNext> {
             timestamp: msg.timestamp,
         };
 
+        if let Some(stop_timestamp) = self.stop_at_timestamp {
+            if metadata.timestamp.timestamp() > stop_timestamp {
+                anyhow::bail!("message is after stop timestamp");
+            }
+        }
+
```

Comment:
I don't think I understand why this is needed. 
Isn't the condition already in`result_to_next_msg`

Reply:
+1

Reply:
sorry, this was an old iteration and i forgot to push, updated